bootstrap/017_query_history.sql

```diff
@@ -166,3 +170,32 @@ BEGIN
         call internal.set_config(:key, 'true');
     end if;
 END;
+
+create or replace procedure internal.update_qtag_day()
+returns varchar
+language sql
+comment = 'materialize qtag_filter for a single day until fully backfilled'
+as
+begin
+
+    let res resultset := (
+        SELECT
+            distinct start_time::date as day
```

Comment:
you can use st_day here and below. it's a partition column. it probably isn't logically different but makes thing clearer.

Reply:
using `st_peroid` which is what you meant I think. Its `datetrunc(day, start_time)`